RHIDP-4684 Update Keycloak configuration instructions (#681)
redhat-developer/red-hat-developers-documentation-rhdh: modules/authentication/proc-enabling-authentication-with-rhsso.adoc

* **Client ID**
* **Client Secret**

.. Configure your {rhsso} realm for performance and security:

> ... Navigate to the **Configure > Realm Settings**.

Reviewer: ```suggestion
... Navigate to the **Configure** > **Realm Settings**.
```

> ... Set the **Access Token Lifespan** to a value greater than 5 min (ideally 10 or 15 minutes) to avoid performance issue caused by unnecessary refresh token requests sent for every API call.

Reviewer: ```suggestion
... Set the **Access Token Lifespan** to a value greater than five minutes (preferably 10 or 15 minutes) to prevent performance issues from frequent refresh token requests for every API call.
```

----
--

> `auth.backstageTokenExpiration`::

Reviewer 2: Note that setting this configuration does not have a impact on the originating performance and security issue, but imo would be still beneficial to include here so that users are aware of the existence of this config.

--

> To change {product-short} token expiration from the default value of one hour.

Reviewer: ```suggestion
To modify the {product-short} token expiration from its default value of one hour, note that this refers to the validity of short-term cryptographic tokens, not the session duration. The expiration value must be set between 10 minutes and 24 hours.
```

> Note that this is not the session duration, but rather the duration that the short-term cryptographic tokens are valid for.

Reviewer: ```suggestion

```

> You cannot set the expiration value lower than 10 minutes or above 24 hours.

Reviewer: ```suggestion

```
